development-defaults.ini

```diff
@@ -113,16 +149,8 @@ no_shirt     = 0
 'Squarewave Lanyard'          = 3
 
 
-
 [enums]
 
-[[badge_status]]
```

Comment:
Yeah, I noticed that and turned deleting back on for now, though implementing "archiving" is one of the next things on my plate for this month.